rcaracaus/desarol
# 168: Layout-Max on header

Steps to test:
1) to to site
2) see header is limited by max-width component

+ Also added min-heights on team-listing icons so they don't jump around as the page loads.

Steps to test:
1) go to team listing page
2) icons should be 150px tall before their images load
